test/integration/kueuectl/resume_test.go

```diff
@@ -111,6 +111,13 @@ var _ = ginkgo.Describe("Kueuectl Resume", ginkgo.Ordered, ginkgo.ContinueOnFail
 						g.Expect(ptr.Deref(createdClusterQueue.Spec.StopPolicy, v1beta1.None)).Should(gomega.Equal(v1beta1.None))
 					}, util.Timeout, util.Interval).Should(gomega.Succeed())
 				})
+
+				ginkgo.By("Delete the ClusterQueue", func() {
```

Comment:
I would suggest to use `util.ExpectClusterQueueToBeDeleted` to don't repeat the code, and wrap inside `ginkgo.DeferCleanup` so that we clean up in case the test itself fails.

Reply:
done